### DISCUSSION TITLE
About token limations.

### DISCUSSION THREAD

**Discussion by rainsoulsrx** [PixArt-alpha/PixArt-alpha#101] [Q&A]
Hi, I notice that in pixart alpha paper, you mentioned 120 tokens are used. But in this page "https://pixart-alpha.github.io/ ", some examples token number is exceed 120. Just trunct to 120?

examples:
"The parametric hotel lobby is a sleek and modern space with plenty of natural light. The lobby is spacious and open with a variety of seating options. The front desk is a sleek white counter with a parametric design. The walls are a light blue color with parametric patterns. The floor is a light wood color with a parametric design. There are plenty of plants and flowers throughout the space. The overall effect is a calm and relaxing space. occlusion, moody, sunset, concept art, octane rendering, 8k, highly detailed, concept art, highly detailed, beautiful scenery, cinematic, beautiful light, hyperreal, octane render, hdr, long exposure, 8K, realistic, fog, moody, fire and explosions, smoke, 50mm f2.8"

**Comment by lawrence-cj** [comment]
Yes, just trunct to 120
